stories/core/TimePicker.stories.tsx

```diff
@@ -0,0 +1,95 @@
+/*---------------------------------------------------------------------------------------------
+ * Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+ * See LICENSE.md in the project root for license terms and full copyright notice.
+ *--------------------------------------------------------------------------------------------*/
+import { action } from '@storybook/addon-actions';
+import { useEffect, useState } from '@storybook/addons';
+import { Meta, Story } from '@storybook/react';
+import React from 'react';
+import {
+  TimePicker,
+  TimePickerProps,
+  IconButton,
+  LabeledInput,
+  PeriodType,
+} from '../../src/core';
+import SvgCalendar from '@itwin/itwinui-icons-react/cjs/icons/Calendar';
+
+export default {
+  title: 'Core/TimePicker',
+  component: TimePicker,
+  argTypes: {
+    onChange: { control: { disable: true } },
+    className: { control: { disable: true } },
+    style: { control: { disable: true } },
+    hourRenderer: {
```

Comment:
Dont like these, but somehow storybook stringified default values and I was getting errors...
Might be connected:
https://github.com/storybookjs/storybook/issues/12455

Reply:
Found the problem: https://github.com/storybookjs/storybook/issues/12098#issuecomment-758153653

I don't think there is a better workaround than what you did

Reply:
Oh btw can you use `args` instead?

`argTypes.defaultValue` will be deprecated soon: https://github.com/storybookjs/storybook/blob/next/MIGRATION.md#deprecated-argtypedefaultvalue

Reply:
moved to args..